fastai/core.py

```diff
@@ -3,37 +3,36 @@
 
 def sum_geom(a,r,n): return a*n if r==1 else math.ceil(a*(1-r**n)/(1-r))
 
+def is_listy(x): return isinstance(x, (list,tuple))
+def is_iter(x): return isinstance(x, collections.Iterable)
+def map_over(x, f): return [f(o) for o in x] if is_listy(x) else f(x)
```

Comment:
@jph00 I rolled this back to using `is_listy` because `collections.Iterable` also ends up catching `np.ndarray`, `torch.Tensor`, and `torch.autograd.Variable` in the net.

Reply:
@mcskinner  see the comments above, I think `sapply` represents better your idea than `map_over`, especially if we extend it to work with dicts and any depths of nested structures.